components/Home/Bonding/BondingList/BondingList.jsx

```diff
@@ -43,6 +44,12 @@ const Container = styled.div`
   }
 `;
 
+const Loader = () => (
+  <Skeleton.Button size="small" style={{ width: '100%' }} active block />
```

Comment:
The conventional way to display loading numbers is simply a '--'. No need to update here, this gets the job done. Just one for the future.